go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -164,7 +164,9 @@ func init() {
 
 	flag.BoolVar(&enableReplicationReporter, "enable_replication_reporter", false, "Use polling to track replication lag.")
 	flag.BoolVar(&currentConfig.EnableOnlineDDL, "queryserver_enable_online_ddl", true, "Enable online DDL.")
-	flag.BoolVar(&currentConfig.SanitizeLogMessages, "sanitize_log_messages", false, "Remove potentially sensitive information in tablet INFO, WARNING, and ERROR log messages such as query parameters.")
+
+	flag.Int64Var(&currentConfig.RowStreamer.MaxTrxHistLen, "vreplication_copy_phase_max_trx_history", 1000000, "The maximum transaction history that can exist on a vstreamer (source) before starting another round of copying rows. This helps to limit the impact on the source tablet.")
+	flag.Int64Var(&currentConfig.RowStreamer.MaxReplLagSecs, "vreplication_copy_phase_max_repl_lag", 43200, "The maximum replication lag (in seconds) that can exist on a vstreamer (source) before starting another round of copying rows. This helps to limit the impact on the source tablet.")
```

Comment:
Should note here that this is for the mysql replication lag on replicas, and not related to the lag reported by vreplication.

Reply:
Good point. That's also why I like adding mysql in the flag name. I'm going to go with those proposed names for now and specify MySQL in the description too.